src/Domain/Quiz/Repository/Interfaces/QuizQuestionAnswersRepository.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App\Domain\Quiz\Repository\Interfaces;
+
+use App\Domain\Quiz\Entity\Answer;
+use App\Presentation\DTO\QuizQuestionAnswerRequestDTO;
+
+interface QuizQuestionAnswersRepository
```

Comment:
```suggestion
interface QuizQuestionAnswersRepositoryInterface
```
ну  Quiz можно опустить так как в неймспейсе есть